### ISSUE TITLE
:Neomake doesn't work in Cargo workspaces (Rust)

### ISSUE THREAD

**Issue by davidpdrsn** [neomake/neomake#1916]
### Expected behavior

In normal library/binary crates `Neomake` works just fine, but in [Cargo workspaces](https://doc.rust-lang.org/book/second-edition/ch14-03-cargo-workspaces.html) it does not. Nothing happens when running `:Neomake`.

### Steps to reproduce

I have made a repo for easy reproduction: https://github.com/davidpdrsn/neomake-cargo-workspace-broken
It contains both a normal crate and one using workspaces.

### Output from (verbose) NeomakeInfo

`:verbose NeomakeInfo`: https://gist.github.com/davidpdrsn/5361fb0fa5c8c5e2f7d17d72ecf4699e

**Comment by m-lima** [comment]
@davidpdrsn This is quite an old issue and I am having the same problem currently. So I'm wondering, did you work something out or find a fix?

**Comment by davidpdrsn** [comment]
@m-lima I don't think so. I haven't used neomake in a very long time. Don't even remember filing this issue tbh. I currently use coc with rust-analyzer and run `cargo build` directly from the command line.
